guides/front_end/vue_quick_start.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Vue3 quick start — Meilisearch documentation
+description: Integrate a search-as-you-type experience into your Vue app.
+---
+
+# Vue3 quick start
+
+## 1. Create a Vue application
+
+```bash
+npm create vue@latest my-app
+```
+
+## 2. Install the library of search components
+
+Navigate to your Vue app and install `vue-instantsearch`, `@meilisearch/instant-meilisearch`, and `instantsearch.css`.
+
+```bash
+npm install vue-instantsearch @meilisearch/instant-meilisearch instantsearch.css
+```
+
+- [Vue InstantSearch](https://github.com/algolia/instantsearch/): front-end tools to customize your search environment
+- [instant-meilisearch](https://github.com/meilisearch/meilisearch-js-plugins/tree/main/packages/instant-meilisearch): Meilisearch client to connect with Vue InstantSearch
+- [instantsearch.css](https://github.com/algolia/instantsearch/tree/master/packages/instantsearch.css) (optional): CSS library to add basic styles to the search components
+
+## 3. Add InstantSearch
+
+Include InstantSearch into `main.js` to include the Vue InstantSearch library.
+
+```js
+import { createApp } from 'vue';
+import App from './App.vue';
+import InstantSearch from 'vue-instantsearch/vue3/es';
+
+const app = createApp(App);
+app.use(InstantSearch);
+app.mount('#app');
+```
+
+## 4. Initialize the search client
+
+Use the following URL and API key to connect to a Meilisearch instance which contains data from Steam video games. Add the code below to the `App.vue` file.
```

Comment:
I think this could benefit from a minor rewrite. We say the "following URL and API key", but these only appear almost at the end of the code sample.

Perhaps a simple way of improving the flow here is to move the info about the URL and key beneath the code sample, as a minor clarification?

Reply:
Could even be a capsule "hey, FYI, these URL and key point to a public instance etc"